Source/WebCore/page/scrolling/AsyncScrollingCoordinator.h

```diff
@@ -75,6 +75,8 @@ class AsyncScrollingCoordinator : public ScrollingCoordinator {
     
     WEBCORE_EXPORT void setMouseIsOverContentArea(ScrollableArea&, bool) override;
     WEBCORE_EXPORT void setMouseMovedInContentArea(ScrollableArea&) override;
+    WEBCORE_EXPORT void setLayerHostingContextIdentifier(ScrollingNodeID, Markable<LayerHostingContextIdentifier>) override;
```

Comment:
Let's call this `setLayerHostingContextIdentifierForFrameHostingNode()`. It's better to use `std::optional<?>` for arguments.

Reply:
Fixed.